drivers/sensor/grove/Kconfig

```diff
@@ -3,16 +3,23 @@
 # Copyright (c) 2015 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config GROVE_SENSORS
+	bool
+	help
+	  Hidden option to indicate if any Grove sensors are enabled.
+
 config GROVE_LIGHT_SENSOR
```

Comment:
Use an `if GROVE_SENSORS` block instead of select

Reply:
Was trying to avoid breaking builds as they would have to enable `CONFIG_GROVE_SENSORS` separately. But this would be better in the long run. So changed the hidden kconfig to a visible one, and all other Grove sensor kconfig are grouped under this new one.